frameworks/Dart/angel3/angel3-mysql.dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM dart:latest
```

Comment:
Can you tie this down to a specific version of dart, please

Reply:
- Removed favicon
- Updated to use official dart docker image 2.18.1